src/binding/fortran/use_mpi_f08/mpi_f08_link_constants.f90

```diff
@@ -24,9 +24,6 @@ module mpi_f08_link_constants
 type(MPI_Status), bind(C, name="MPIR_F08_MPI_STATUS_IGNORE_OBJ"), target :: MPI_STATUS_IGNORE
 type(MPI_Status), dimension(1), bind(C, name="MPIR_F08_MPI_STATUSES_IGNORE_OBJ"), target :: MPI_STATUSES_IGNORE
 
-type(c_ptr), bind(C, name="MPIR_C_MPI_STATUS_IGNORE") :: MPIR_C_MPI_STATUS_IGNORE
-type(c_ptr), bind(C, name="MPIR_C_MPI_STATUSES_IGNORE") :: MPIR_C_MPI_STATUSES_IGNORE
-
 ! Though these two variables are required by MPI-3 Standard, they are not used in MPICH
 type(c_ptr), bind(C, name="MPI_F08_STATUS_IGNORE")   :: MPI_F08_STATUS_IGNORE   ! Point to MPI_STATUS_IGNORE
 type(c_ptr), bind(C, name="MPI_F08_STATUSES_IGNORE") :: MPI_F08_STATUSES_IGNORE ! Point to MPI_STATUSES_IGNORE
```

Comment:
Are MPI_F08_STATUS_IGNORE and MPI_F08_STATUSES_IGNORE broken similar to MPIR_C_MPI_STATUS_IGNORE and MPIR_C_MPI_STATUSES_IGNORE?  I know MPICH does not use them, but what if the user does?

Reply:
Both `MPI_F08_STATUS_IGNORE` and `MPI_F08_STATUSES_IGNORE ` are constants (`1`) defined in `mpi.h` and they are statically initialized in `src/binding/fortran/use_mpi_f08/cdesc.c`. It worked, but it relied on the value being constants defined in `mpi.h`.

`MPI_F08_STATUS_IGNORE ` and `MPI_F08_STATUSES_IGNORE ` works the same way as `MPIR_C_MPI_UNWEIGHTED` and `MPIR_C_MPI_WEIGHTS_EMPTY`, but since the symbols on C side is missing `MPICH_API_PUBLIC`, thus not exposed, it is likely broken. But there is nothing checks it.